versions/3.1.1.md

```diff
@@ -157,7 +157,7 @@ The formats defined by the OAS are:
 `integer` | `int64` | signed 64 bits (a.k.a long)
 `number` | `float` | |
 `number` | `double` | |
-`string` | `password` | A hint to UIs to obscure input.
+`string` | `password` | A hint to obscure/redact the value.
```

Comment:
So based on [that comment](https://github.com/OAI/OpenAPI-Specification/pull/3413#issuecomment-1910618000) recommended wording is:

```suggestion
`string` | `password` | A hint to obscure the value.
```

and the same change made to relevant [the registry file](https://github.com/OAI/OpenAPI-Specification/blob/gh-pages/registries/_format/password.md?plain=1#L4).